Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -876,28 +867,29 @@ private function whenDimensionSpacePointWasMoved(DimensionSpacePointWasMoved $ev
         // hierarchy relations for this query. Then, we update the Hierarchy Relations.
 
         // 1) originDimensionSpacePoint on Node
-        $rel = $this->getDatabaseConnection()->executeQuery(
-            'SELECT n.relationanchorpoint, n.origindimensionspacepointhash
-                 FROM ' . $this->tableNames->node() . ' n
-                 INNER JOIN ' . $this->tableNames->hierarchyRelation() . ' h
-                    ON h.childnodeanchor = n.relationanchorpoint
-
-                 AND h.contentstreamid = :contentStreamId
-                 AND h.dimensionspacepointhash = :dimensionSpacePointHash
-                 -- find only nodes which have their ORIGIN at the source DimensionSpacePoint,
-                 -- as we need to rewrite these origins (using copy on write)
-                 AND n.origindimensionspacepointhash = :dimensionSpacePointHash
-            ',
-            [
+        $selectRelationsStatement = <<<SQL
+            SELECT n.relationanchorpoint
+            FROM {$this->tableNames->node()} n
+            INNER JOIN {$this->tableNames->hierarchyRelation()} h
+                ON h.childnodeanchor = n.relationanchorpoint
+                AND h.contentstreamid = :contentStreamId
+                AND h.dimensionspacepointhash = :dimensionSpacePointHash
+                -- find only nodes which have their ORIGIN at the source DimensionSpacePoint,
+                -- as we need to rewrite these origins (using copy on write)
+                AND n.origindimensionspacepointhash = :dimensionSpacePointHash
+        SQL;
+        try {
+            $relationAnchorPoints = $this->dbal->fetchFirstColumn($selectRelationsStatement, [
                 'dimensionSpacePointHash' => $event->source->hash,
                 'contentStreamId' => $event->contentStreamId->value
-            ]
-        );
-        while ($res = $rel->fetchAssociative()) {
-            $relationAnchorPoint = NodeRelationAnchorPoint::fromInteger($res['relationanchorpoint']);
+            ]);
```

Comment:
here something more happened than just cleanup ;) I suspect you mean this bugfix?

Reply:
what do you mean? It's the same SQL with slightly different indentations

Reply:
ah sorry i meant that we dont select ` n.origindimensionspacepointhash` anymore

Reply:
well because we dont need it 👍

Reply:
Yes. Eagle-eye, respect ;)
But you're correct, it was never used